plugins/deep-load.ts

```diff
@@ -44,10 +44,12 @@ export default ({ app }) => {
       const sectionBasepath =
         typeof section.basepath !== 'undefined' ? section.basepath : basepath
 
-      section.collections[aCollection] =
-        await Promise.all(section.collections[aCollection].map(
-          path => import(`~/content/${sectionBasepath}${path}`)
-        ))
+      if (typeof section.collections[aCollection][0] === 'string') {
+        section.collections[aCollection] =
+          await Promise.all(section.collections[aCollection].map(
+            path => import(`~/content/${sectionBasepath}${path}`)
+          ))
+      }
```

Comment:
Instead of checking the type, I would prefer addressing the root problem. It happens that we are modifying the imported object at [line 31](https://github.com/qiskit-community/community.qiskit.org/pull/89/files#diff-c43f0d9229e2acc653766d7e9e44ebb5R31) instead of a copy. We can solve this creating a copy of the collection before replacing the strings with the actual objects or preventing the same TOC for being loaded twice. I think the latter is easier.